sweepai/utils/docker_utils.py

```diff
@@ -0,0 +1,29 @@
+import requests
+import json
+import subprocess
+
+def get_docker_hub_version():
+    response = requests.get('https://hub.docker.com/v2/repositories/sweepai/sweep/tags')
+    data = json.loads(response.text)
+    for result in data['results']:
+        last_pushed = result['last_pushed']
+        if last_pushed:
+            return last_pushed
+    return 'No version found'
+
+def get_local_docker_version():
+    result = subprocess.run(['docker', 'images', 'sweepai/sweep', '--format', '{{.CreatedAt}}'], stdout=subprocess.PIPE)
+    version = result.stdout.decode('utf-8').strip()
+    return version if version else 'No version found'
+
+
+def get_docker_version_badge():
+    try:
+        hub_version = get_docker_hub_version()
+        local_version = get_local_docker_version()
+        if hub_version == local_version:
+            return 'Up-to-date'
```

Comment:
instead of these use
Label, message and color seperated by a dash -. For example:
any text: you like -https://img.shields.io/badge/any_text-you_like-blue